src/spells/CESFork_GoerliRwaSpell.sol

```diff
@@ -89,27 +89,27 @@ contract SpellAction {
     // The contracts in this list should correspond to MCD core contracts, verify
     // against the current release list at:
     //     https://changelog.makerdao.com/releases/goerli/latest/contracts.json
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0x6a4D20288D43bDe175842a78e7C30381045550f3);
 
     /*
-        OPERATOR: 0xA5Eee849FF395f9FA592645979f2A8Af6E0eF5c3
+        OPERATOR: 0x974217c5721f3CCefCE3A96A2d05ff08C250B267
```

Comment:
Whose address is this?

Reply:
The one who authorized in output conduit

Reply:
I mean who controls this?

Reply:
I assume it's an address @nduchak controls? Maybe add the priv key to our internal doc as well.